### PR TITLE
this.javascript

### PR DESCRIPTION
**Pull request by amyrlam**

### CODE REVIEW

File: source/2019-08-09-the-ember-times-issue-110.md [hunk 1]
```diff
@@ -40,13 +42,11 @@ For a deeper dive you can also check out [@skaterdav85](https://github.com/skate
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [This.JavasSript: State of Frameworks 📜](https://www.thisdot.co/events/this-javascript-state-of-frameworks-08-2019)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+This.JavaScript is an online livestreamed event on **Tuesday, August 13th** - [RSVP to attend](https://www.thisdot.co/events/this-javascript-state-of-frameworks-08-2019) and watch it with your teams at work!
```
Comment (jayjayjpg) [inline source/2019-08-09-the-ember-times-issue-110.md]:
```suggestion
This.JavaScript is an online livestreamed event on **Tuesday, August 13th** - [RSVP to attend](https://www.thisdot.co/events/this-javascript-state-of-frameworks-08-2019) and watch it with your teams at work!
```

File: source/2019-08-09-the-ember-times-issue-110.md [hunk 2]
```diff
@@ -40,13 +42,11 @@ For a deeper dive you can also check out [@skaterdav85](https://github.com/skate
...
-## [Section Title in Title Case 🐹](#section-url)
+## [This.JavasSript: State of Frameworks 📜](https://www.thisdot.co/events/this-javascript-state-of-frameworks-08-2019)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+This.JavaScript is an online livestreamed event on **Tuesday, August 13th** - [RSVP to attend](https://www.thisdot.co/events/this-javascript-state-of-frameworks-08-2019) and watch it with your teams at work! 
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+The State of Frameworks event is an opportunity for folks in the JS community from different frameworks and libraries to learn from each other and connect. You'll learn about upcoming releases, latest improvements, and community initiatives. [@jenweber](https://github.com/jenweber) will be representing Ember and speaking about Octane! She'll be joined by [@yyx990803](https://github.com/yyx990803) of Vue, [@mhdawson](https://github.com/mhdawson) of Node.js, [@marvinhagemeister](https://github.com/marvinhagemeister) of Preact, [@mhartington](https://github.com/mhartington) of Ionic, and [@mgechev](https://github.com/mgechev) of Angular!
```
Comment (jayjayjpg) [inline source/2019-08-09-the-ember-times-issue-110.md]:
```suggestion
The State of Frameworks event is an opportunity for folks in the JS community from different frameworks and libraries to learn from each other and connect. You'll learn about upcoming releases, latest improvements, and community initiatives. [@jenweber](https://github.com/jenweber) will be representing Ember and speaking about Octane! She'll be joined by [@yyx990803](https://github.com/yyx990803) of Vue, [@mhdawson](https://github.com/mhdawson) of Node.js, [@marvinhagemeister](https://github.com/marvinhagemeister) of Preact, [@mhartington](https://github.com/mhartington) of Ionic, and [@mgechev](https://github.com/mgechev) of Angular!
```

**Review by jayjayjpg — APPROVED** [review]
This reads great! I left two more suggestions to remove trailing whitespaces, but otherwise this should be good to go - thank you for another :sparkles: writeup this week!

**Comment by amyrlam** [comment]
@jessica-jordan thx, adding a commit to turn this rule off

**Pull request merged**